src/Home_excercise_5.java

```diff
@@ -0,0 +1,116 @@
+            System.out.print("Enter your symbol: ");
+            String yourCharacter = sc1.nextLine();
+            int amountSymbols = findSymbolOccurance1(yourString,yourCharacter);
+            System.out.println("Total same symbols: "+amountSymbols);
+            // int countSymbol = findSymbolOccurance2(yourString,yourCharacter);//It's my the second version, but
+            //something wrong, please HELP!!!
+            // System.out.println("Amount of symbol: "+countSymbol);
+
+            Scanner sc2 = new Scanner(System.in);
+            System.out.print("Enter your next text: ");
+            String nextText = sc2.nextLine();
+            Scanner sc3 = new Scanner(System.in);
+            System.out.print("Enter your check text: ");
+            String checkText = sc3.nextLine();
+            int indexFirstSymbol = findWordPosition(nextText,checkText);
+            System.out.println("The index of the first symbol: "+indexFirstSymbol);
+            System.out.println("The revers your text: "+stringReverse(nextText));
+            System.out.println("Your text is Palindrome: " + isPalindrome(nextText));
+
+            Random random = new Random();
+            String [] secretWord = {"apple", "orange", "lemon", "banana", "apricot", "avocado", "broccoli", "carrot",
+                    "cherry", "garlic", "grape", "melon", "leak", "kiwi", "mango", "mushroom", "nut", "olive", " pea",
+                    "peanut", "pear", "pepper", "pineapple", "pumpkin", "potato"};
+            Scanner sc4 = new Scanner(System.in);
+
+            int indexElement = random.nextInt(secretWord.length);
+            String randomWord = secretWord[indexElement];
+
+            //System.out.println(indexElement);
+            //System.out.println(secretWord[indexElement]);
+            int account = 1;
+            while ((account-1)<randomWord.length()) {
+                System.out.println("Round "+account);
+                System.out.print("Enter your word (vegetable or fruit): ");
+                String yourRandomWord = sc4.nextLine().trim().toLowerCase();
+                if (yourRandomWord.equals(randomWord)) {
+                    System.out.println("Congratulation!!! You are Winner! ");
+                    break;
+                } else {
+                    System.out.println("Wrong! Try again!");
+                    account++;
+                    String checkWorld = "###############";
+                    System.out.println(randomWord.substring(0,(account-1))+checkWorld.substring(account-1));
+                }
+            }
+            System.out.println("Unfortunately, you are lose! Don't wary! Next time you get lucky!");
+        }
+
+
+        public static int findSymbolOccurance1(String yourString,String yourCharacter) {
+            int amountChracter = 0;
+            for (int i = 0; i < yourString.length(); i++) {
+                if ((String.valueOf(yourString.charAt(i))).equals(yourCharacter)){
```

Comment:
There is no need to use `String.valueOf()` for chars comparison. You can simply do it like this: `str.charAt(i) == anotherCharacter`.